src/main/templates/ratings.hbs

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  {{>header}}
+  <!-- Main content starts - do not edit above this line -->
+    <table id="sorttable" class="table table-small table-hover display">
+      <thead class="thead-light">
+        <tr>
+          <th scope="col" class="align-text-top">Region</th>
+          <th scope="col" class="align-text-top" width="10%">DCNC Code <small>(For CTT)</small></th>
```

Comment:
```suggestion
          <th scope="col" class="align-text-top" width="10%">DCNC Code<br/><small>(For CTT)</small></th>
```

Reply:
<br>'s are not added on the table headers (same as language and territory) to allow for a single line header height on full page. The parenthetical descriptions break properly on the page width for ISDCF page embeds where required.

Reply:
![image](https://user-images.githubusercontent.com/4871350/98845688-ed9dac00-2402-11eb-831e-df1808fd038f.png)

Ok. The observable problem is that the column width seem to be determined by the heading and not actual contents, which makes them hard to read and inefficient.

Reply:
Noted, let me see how it works with all of the headers using breaks for the parenthetical info. I'd rather be consistent on all pages.

Reply:
Yep, works well enough with them added everywhere, was able to remove some of the width definitions as a result. in 1e9af3f